proto/autoscaling/kubernetes/autoscaling.proto

```diff
@@ -7,12 +7,20 @@ package datadog.autoscaling.kubernetes;
 import "google/protobuf/timestamp.proto";
 import "proto/deps/github.com/chrusty/protoc-gen-jsonschema/options.proto";
 
+enum ErrorCode {
+  ERROR_UNSPECIFIED = 0;
+  ERROR_HEALTH = 10;
+  ERROR_INTERNAL = 20;
+  ERROR_CONFIG = 30;
+  ERROR_NODATA = 40;
+}
+
 // Error represents an error message
 message Error {
   option (protoc.gen.jsonschema.message_options).ignore = true;
 
-  optional int32 code    = 1;
-  string         message = 2;
+  optional ErrorCode code    = 1;
```

Comment:
question: will previous clients only send the correct enum values? iirc [enum types are represented compatibly](https://groups.google.com/g/protobuf/c/MTkIHnM_QVg) on the wire as int32 but if they are sending incorrect values we might hit a index error / unknown field from the deserialization code